### PR TITLE
Stopping names from appearing in Question Titles

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
This PR was raised to stop names from appearing in question page titles. 

It also includes a fix for content pages - they now use the content title where they used to use the group title within the page title.

### How to review 
Run a schema and check that:
- 'Question' Page titles no longer contain names.
- 'Content' pages now use the content title in the page title.
- Other page titles are correct
- Tests pass

### CODE REVIEW

**Comment by berroar** [comment]
[Names are appearing in page titles in question](https://trello.com/c/z5qR48eU/3316-names-are-appearing-in-page-titles-in-question)

File: app/views/handlers/content.py [hunk 1]
```diff
@@ -24,6 +24,22 @@ def _render_block(self, block_id):
...
 
+        self.page_title = self._get_page_title(transformed_block)
+
         return self.placeholder_renderer.render(
             transformed_block, self._current_location.list_item_id
         )
+
+    def _get_page_title(self, transformed_block):
+        if 'content' in transformed_block:
+            if type(transformed_block['content']['title']) is str:
```
Comment (MebinAbraham) [inline app/views/handlers/content.py]: Can we use `isinstance` instead?. Same comment for `question.py`
``` py
    def _get_page_title(self, transformed_block):
        content = transformed_block.get('content')
        if content:
            if isinstance(content['title'], str):
                content_title = content['title']
            else:
                content_title = content['title']['text']

            page_title = f'{content_title} - {self._schema.json["title"]}'

            return safe_content(page_title)
```

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Apart from my 1 minor comment, all looks good to me. 👍

**Review by JamesGardiner — APPROVED** [review]
LGTM once @MebinAbraham 's changes have been addressed

**Review by JamesGardiner — CHANGES_REQUESTED** [review]
case_id needs to be added to the metadata in the failing test schema

**Review by MebinAbraham — APPROVED** [review]

File: data-source/json/test_interstitial_page_title.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
```
Comment (JamesGardiner) [inline data-source/json/test_interstitial_page_title.json:30]: You can add this here:

```JSON
{
    "name": "case_id",
    "type": "string"
}
```

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**